{"name": "test_update_user_info[Update User Info - Firstname & Lastname are Above Max Value]", "status": "failed", "statusDetails": {"message": "AssertionError: FAILURE: \u001b[31massert 200 == 400\ntests\\test_api.py:181 in test_update_user_info() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1", "trace": "FAILURE: \u001b[31massert 200 == 400\ntests\\test_api.py:181 in test_update_user_info() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1"}, "attachments": [{"name": "log", "source": "529be339-8411-405a-bc4d-a3cc7d5d2dfd-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'Case': 'Update User Info - Firstname & Lastname are Above Max Value', 'Test Type': 'Negative', 'Request_Body': {'firstName': 'WwqOZzewCSEMgTGtpZeHmYVAfZZgZHQPsFrSVgcvOanboDckGla', 'lastName': 'FXJcbCVRnAselDsdSxiueYirXtvuFKfJFgFqcyKZrIaoLFvnrcp'}}"}], "start": 1670168778083, "stop": 1670168778794, "uuid": "1ac78f32-0370-4848-9328-9367087e97d3", "historyId": "1deaa3f95258cdb6d50018eba0e40768", "testCaseId": "9b22326ca1aa84dd7b5876f09937fe1e", "fullName": "tests.test_api.TestApi#test_update_user_info", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "DESKTOP-EGG762M"}, {"name": "thread", "value": "11972-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}